### ISSUE TITLE
Crash in shepard__s_crossing

### ISSUE THREAD

**Issue by sum2012** [sum2012/ppsspp-problem#45]
@Honcharuk  
Only talk in here.I take a search that shepard__s_crossing.cso is a patched iso.
Can you try download iso in 
http://www.emuparadise.me/PSP_ISOs/Shepherd's_Crossing_(USA)/158544
?

**Comment by Honcharuk** [comment]
No, i have iOS 9 on my iPhone 5 (experimental crashes too) and 8.4 on iPad 3 (stable build crashes, before all be good and cso not change)

Cso from my psp 3008:)
https://yadi.sk/d/vXxKE-sRkTkbc

Honcharuk

**Comment by sum2012** [comment]
Can you upload your Cso into
http://drive.google.com/ ?

**Comment by Honcharuk** [comment]
Yes, https://drive.google.com/open?id=0Bw56O02t8CzsclptbTk1RWM5LW8

2015-11-15 9:29 GMT+02:00 sum notifications@github.com:

**Comment by sum2012** [comment]
No problem on me windows and Android
Seem only ios issue
![1](https://cloud.githubusercontent.com/assets/2177532/11169048/b89ad64c-8be4-11e5-86ec-b6562296a50c.jpg)

**Comment by Honcharuk** [comment]
on ios only intro video and in menu crashes.
i not have a lot time with mi computer.

2015-11-15 16:04 GMT+02:00 sum notifications@github.com:

**Comment by Honcharuk** [comment]
[image: Встроенное изображение 1] in ppsspp.ini "Language = ru_RU" but
steel English lang.

2015-11-15 16:28 GMT+02:00 Максим Гончарук 2.6makc@gmail.com:

**Comment by Honcharuk** [comment]
And i have\* Logitech attack 3\* joystick.

Can I play with him ?

2015-11-15 16:39 GMT+02:00 Максим Гончарук 2.6makc@gmail.com:

**Comment by sum2012** [comment]
I don't have ios device so that I cannot help
